src/bindings/gsettings/CMakeLists.txt

```diff
@@ -48,8 +48,8 @@ if (CMAKE_VERSION GREATER 3.3)
 	pkg_get_variable (GIO_MODULE_DIR gio-2.0 giomoduledir)
 else ()
 	execute_process (
-		COMMAND PKG_CONFIG_EXECUTABLE --variable=giomoduledir
-		OUTPUT GIO_MODULE_DIR
+		COMMAND PKG_CONFIG_EXECUTABLE --variable=giomoduledir gio-2.0
+		OUTPUT_VARIABLE GIO_MODULE_DIR
```

Comment:
Did you try it? I do not think that `PKG_CONFIG_EXECUTABLE` is a command that will be found. (Variable is not dereferenced) I think its better that we remove the if (CMAKE_VERSION). It will make it easier for you to spot such problems.